libbeat/cloudid/cloudid.go

```diff
@@ -157,9 +158,9 @@ func OverwriteSettings(cfg *common.Config) error {
 		return nil
 	}
 
-	logp.Debug("cloudid", "cloud.id: %s, cloud.auth: %s", cloudID, cloudAuth)
+	logger.Debugf("cloudid", "cloud.id: %s, cloud.auth: %s", cloudID, cloudAuth)
```

Comment:
the global logger does not support a selector. Instead the 'selector' is the loggers name.

`logger.Debugf("cloudid", "cloud.id: %s, cloud.auth: %s", cloudID, cloudAuth)`

should become

`logger.Debugf("cloud.id: %s, cloud.auth: %s", cloudID, cloudAuth)`